### PR TITLE
[DO NOT SUBMIT] Add more logging to abort.

### PR DESCRIPTION
**Pull request by mikelehen**
Not for submission. Just suggesting this to possibly help debug https://github.com/firebase/firebase-js-sdk/issues/1642

### CODE REVIEW

File: packages/firestore/src/local/simple_db.ts [hunk 1]
```diff
@@ -312,7 +314,8 @@ export class SimpleDbTransaction {
     this.transaction.oncomplete = () => {
       this.completionDeferred.resolve();
     };
-    this.transaction.onabort = () => {
+    this.transaction.onabort = (event) => {
```
Comment (larssn) [inline packages/firestore/src/local/simple_db.ts]: Is this correct? `event` is not getting used.

Comment (mikelehen) [reply packages/firestore/src/local/simple_db.ts]: Oh, that could be reverted...  I was just hacking this together real quick and am not planning to submit it as-is. 😄

Comment (larssn) [reply packages/firestore/src/local/simple_db.ts]: Alright, just wanted to be sure where you'd want the error from. 😅 

I'm pushing a new release with your suggestions here; so we should have some kind of data fairly soon.

Comment (mikelehen) [reply packages/firestore/src/local/simple_db.ts]: Ah, fantastic!  I think while writing this I was wondering the same thing (if there was error information in the `event`) but discovered that there was not.

**Pull request closed without merging**